src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/TeamPlanControls/PriceCallout/PriceCallout.tsx

```diff
@@ -0,0 +1,84 @@
+import { useParams } from 'react-router-dom'
+
+import { useAvailablePlans } from 'services/account'
+import {
+  findTeamPlans,
+  formatNumberToUSD,
+  isAnnualPlan,
+  Plans,
+} from 'shared/utils/billing'
+import { calculatePriceTeamPlan } from 'shared/utils/upgradeForm'
+import Icon from 'ui/Icon'
+
+interface PriceCalloutProps {
+  newPlan: string
+  seats: number
+  setValue: (x: string, y: string) => void
+}
+
+const PriceCallout: React.FC<PriceCalloutProps> = ({
+  newPlan,
+  seats,
+  setValue,
+}) => {
+  const { provider, owner } = useParams<{ provider: string; owner: string }>()
```

Comment:
I'd made this component in my previous PR, so I just added it here to let it be in the meantime